src/puppetlabs/puppetdb/command.clj

```diff
@@ -111,12 +181,15 @@
   its id."
   [mq-connection :- mq/connection-schema
    mq-endpoint :- s/Str
-   raw-command :- s/Str
-   uuid :- (s/maybe s/Str)]
+   raw-command :- (s/cond-pre s/Str (Class/forName "[B"))
```

Comment:
Why do we want `s/cond-pre` instead of `s/either`? The latter is easier to read, and I don't think we need the ordered matching that `s/cond-pre` claims to provide.